CMakeLists.txt

```diff
@@ -78,25 +80,25 @@ function(set_finufft_options target)
     enable_asan(${target})
 
     target_compile_options(${target} PRIVATE SHELL:$<$<CONFIG:Release,RelWithDebInfo>:${FINUFFT_ARCH_FLAGS}>)
-    if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
         # Enable cx-limited-range on supported compilers
         target_compile_options(${target} PRIVATE $<$<CONFIG:Release,RelWithDebInfo>:-fcx-limited-range>)
-    endif()
+    endif ()
 
     target_include_directories(${target} PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
-    if(FINUFFT_USE_OPENMP)
+    if (FINUFFT_USE_OPENMP)
         target_link_libraries(${target} PRIVATE OpenMP::OpenMP_CXX)
-    else()
-        if(CMAKE_CXX_COMPILER_ID IN_LIST FINUFFT_GNU_LIKE_COMPILERS)
+    else ()
+        if (CMAKE_CXX_COMPILER_ID IN_LIST FINUFFT_GNU_LIKE_COMPILERS)
             # OpenMP disabled, suppress unknown pragma warnings to avoid spam.
             target_compile_options(${target} PRIVATE "-Wno-unknown-pragmas")
-        endif()
-    endif()
+        endif ()
+    endif ()
 endfunction()
 
 # Main finufft libraries
 add_library(finufft_f32 OBJECT ${FINUFFT_PRECISION_DEPENDENT_SOURCES})
-target_compile_definitions(finufft_f32 PRIVATE SINGLE)
+target_compile_definitions(finufft_f32 PRIVATE -DSINGLE=1)
```

Comment:
I guess we don't need define a value for SINGLE, `target_compile_definitions(finufft_f32 PRIVATE SINGLE)` should be good.

Reply:
@DiamonDinoia I guess this is the new commits.

Reply:
The change has been reverted.

Reply:
The change has been reverted.